woocommerce/woocommerce-ios
# [Manual Taxes M2] Populate UI with fetched tax rates

Closes: #10612 


## Description

With this PR we implement the UI population with the fetch tax rates. We also added pagination to the view and the logic to show the bottom notice when all pages are loaded. Furthermore, we tweaked the UI to add some missing parts and extract some views to their own structs.  Apart from that, we fixed some elements in Yosemite and Networking to ensure the right functioning of this feature.
Please note that the empty state is not implemented in this PR to avoid making it even bigger. It will be part of one of the next PRs once the design is finalized.

## Testing instructions


With a store with tax rates (please let me know if you want to use one with many tax rates added)

1. Go to Orders
2. Tap on + to create a new order
3. Tap on Set New Tax Rate
4. Make sure that pagination and pull to refresh work fine. Please also check that the _Edit Tax Rates in Admin_ notice is shown at the bottom

## Screenshots


### Many Pages

https://github.com/woocommerce/woocommerce-ios/assets/1864060/6fb83be4-8744-4a7e-a8cc-8fd09605d8d4

### One page

https://github.com/woocommerce/woocommerce-ios/assets/1864060/8a160c7b-6305-492c-94db-80cfc043918c

---
